src/Query/Wildcard.php

```diff
@@ -5,33 +5,56 @@
 /**
  * Wildcard query.
  *
- * @author Nicolas Ruflin <spam@ruflin.com>
```

Comment:
Removed by accident, will restore it

Reply:
Does not matter TBH. I think it is nice to give "credit" here to contributors, but my name should probably be remove in most places.

Reply:
About contributors: what about having a link to https://github.com/ruflin/Elastica/graphs/contributors ?
But this is a topic for another PR :)